source/matrix_free/evaluation_template_face_factory.cc

```diff
@@ -18,9 +18,7 @@
 DEAL_II_NAMESPACE_OPEN
 
 #define SPLIT_INSTANTIATIONS_COUNT 6
-#ifndef SPLIT_INSTANTIATIONS_INDEX
-#  define SPLIT_INSTANTIATIONS_INDEX 0
-#endif
+#define SPLIT_INSTANTIATIONS_INDEX 0
```

Comment:
I should add that I removed the `#ifndef ... #endif` here as well (and also in all of the other "first" instantiation files). Having forgotten how the split instantiations work, I found the `#ifndef ... #endif` part particularly confusing because it is only understandable if you remember that there are *other* files that `#include` this `.cc` file -- which is easy to forget because it's an uncommon approach.